packages/vuepress-theme-odyssey/styles/docskit/_coin.scss

```diff
@@ -0,0 +1,21 @@
+.docskit-coin {
```

Comment:
Nit: coin is confusing here, we use the term elsewhere to denote a visual image for a component and/or section. Consider renaming to something else.

Reply:
renamed to "marker". "badge" or "counter" might be more apt, but i'd like to keep it generic until we revisit similar UI for ODS.